mehari-python/mehari/__init__.py

```
@@ -109,3 +165,45 @@ def annotate(
+        """
+        parsed_variants = []
+
+        for var in variants:
+            if isinstance(var, str):
+                try:
+                    c, p, r, a = var.split(":")
+                    parsed_variants.append((c, int(p), r, a))
+                except ValueError:
+                    raise ValueError(
+                        f"Invalid format '{var}'. Expected 'chr:pos:ref:alt'"
+                    )
```

Ruff (0.15.6)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)